zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java

```diff
@@ -98,7 +98,10 @@ public void addViewControllers(ViewControllerRegistry registry) {
       // could split the collector's CORS policy into a different property, still allowing POST
       // with content-type by default.
       .allowRequestMethods(HttpMethod.GET, HttpMethod.POST)
-      .allowRequestHeaders(HttpHeaderNames.CONTENT_TYPE);
+      .allowRequestHeaders(HttpHeaderNames.CONTENT_TYPE,
```

Comment:
@trustin fyi in armeria repo I mentioned something about different modules needing to collaborate on CORS config. since this is now baked into zipkin server directly now, there is minimal impact to us.